### ISSUE TITLE
CA "Exhaustive Training" (Promotion of a soldier) way too short.

### ISSUE THREAD

**Issue by bstar1million** [WOTCStrategyOverhaul/CovertInfiltration#353]
Difficulty: Legend
Duration: ~2 days

Should be at least 8-10 days imo. Could be capped to soldier below rank X or scale in duration with soldier rank.

**Comment by cannonfodder1** [comment]
Yeah I forgot to give it a base duration in #335 will fix for 1.1
